### ISSUE TITLE
socket.gaierror: [Errno 4] Non-recoverable failure in name resolution

### ISSUE THREAD

**Issue by jmatsushita** [n-riesco/ijavascript#70]
Hi there,

I get the following error on OSX El Capitan.
```
$ ijs
[TerminalIPythonApp] WARNING | Subcommand `ipython notebook` is deprecated and will be removed in future versions.
[TerminalIPythonApp] WARNING | You likely want to use `jupyter notebook` in the future
Traceback (most recent call last):
  File "/usr/local/bin/ipython", line 11, in <module>
    sys.exit(start_ipython())
  File "/usr/local/lib/python2.7/site-packages/IPython/__init__.py", line 119, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/traitlets/config/application.py", line 595, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/usr/local/lib/python2.7/site-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/IPython/terminal/ipapp.py", line 296, in initialize
    super(TerminalIPythonApp, self).initialize(argv)
  File "<decorator-gen-7>", line 2, in initialize
  File "/usr/local/lib/python2.7/site-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/IPython/core/application.py", line 398, in initialize
    self.parse_command_line(argv)
  File "/usr/local/lib/python2.7/site-packages/IPython/terminal/ipapp.py", line 291, in parse_command_line
    return super(TerminalIPythonApp, self).parse_command_line(argv)
  File "<decorator-gen-4>", line 2, in parse_command_line
  File "/usr/local/lib/python2.7/site-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/traitlets/config/application.py", line 488, in parse_command_line
    return self.initialize_subcommand(subc, subargv)
  File "/usr/local/lib/python2.7/site-packages/IPython/core/application.py", line 208, in initialize_subcommand
    return super(BaseIPythonApplication, self).initialize_subcommand(subc, argv)
  File "<decorator-gen-3>", line 2, in initialize_subcommand
  File "/usr/local/lib/python2.7/site-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/traitlets/config/application.py", line 426, in initialize_subcommand
    self.subapp.initialize(argv)
  File "<decorator-gen-115>", line 2, in initialize
  File "/usr/local/lib/python2.7/site-packages/traitlets/config/application.py", line 74, in catch_config_error
    return method(app, *args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/notebook/notebookapp.py", line 1071, in initialize
    self.init_webapp()
  File "/usr/local/lib/python2.7/site-packages/notebook/notebookapp.py", line 919, in init_webapp
    self.http_server.listen(port, self.ip)
  File "/usr/local/lib/python2.7/site-packages/tornado/tcpserver.py", line 126, in listen
    sockets = bind_sockets(port, address=address)
  File "/usr/local/lib/python2.7/site-packages/tornado/netutil.py", line 155, in bind_sockets
    0, flags)):
socket.gaierror: [Errno 4] Non-recoverable failure in name resolution
```

Here's my config.
```
(ijs)jBook:ijs jun$ ijs --version
5.0.11-beta.5
(ijs)jBook:ijs jun$ npm version
{ npm: '3.9.0',
  ares: '1.10.1-DEV',
  http_parser: '2.5.2',
  icu: '56.1',
  modules: '46',
  node: '4.4.2',
  openssl: '1.0.2g',
  uv: '1.8.0',
  v8: '4.5.103.35',
  zlib: '1.2.8' }
(ijs)jBook:ijs jun$ ipython --version
4.2.1
```

**Comment by jmatsushita** [comment]
Starting with `ijs --ip=0.0.0.0` solved it.

**Comment by n-riesco** [comment]
I believe 48520b68bf9a699d7b321925356dcd2f4a2a5016 fixes this issue (just released as IJavascript@5.0.11).
